### PR TITLE
Added benchmarks to key_value_store example

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: examples/key_value_store/benchmark/benchmark.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import { createActor as createActorMotoko } from '../dfx_generated/motoko';
+import { createActor as createActorRust } from '../dfx_generated/rust';
+
+const MEDIUM_STING: String = `
+    This is a medium sized string. It's a little bit longer than something you
+    would want to have in line, even though it's much shorter than the long
+    block of Lorem Ipsum below.
+`;
+
+const MEDIUM_STRING_VARIENT = MEDIUM_STING + 'v2';
```
Comment (lastmjs) [inline examples/key_value_store/benchmark/benchmark.ts:12]: `VARIENT` should be spelled `VARIANT`

File: examples/key_value_store/benchmark/benchmark.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem
+    sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit
+    amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora
+    incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad
+    minima veniam, quis nostrum exercitationem ullam corporis suscipit
+    laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum
+    iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae
+    consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?`;
+
+const LONG_STRING_VARIENT = LONG_STING + 'v2';
```
Comment (lastmjs) [inline examples/key_value_store/benchmark/benchmark.ts:34]: `VARIENT` should be spelled `VARIANT`

File: examples/key_value_store/canisters/rust/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+version = "0.0.0"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+ic-cdk = "0.5.2"
+ic-cdk-macros = "0.5.2"
+candid = "0.7.14"
```
Comment (lastmjs) [inline examples/key_value_store/canisters/rust/Cargo.toml:12]: Do you need candid? And no newline

File: examples/key_value_store/canisters/motoko/main.mo [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      wasm_including_prelude = Prim.performanceCounter(0);
+    };
+  };
+  //#endregion
+
+  type Store = HashMap.HashMap<Text, Text>;
+
+  let store: Store = HashMap.HashMap(10, Text.equal, Text.hash);
+
+  public func get(key: Text): async Text {
```
Comment (lastmjs) [inline examples/key_value_store/canisters/motoko/main.mo:30]: This function signature does not match Azle and Rust, it should return an optional Text

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: examples/key_value_store/canisters/rust/Cargo.toml (hunk 3 above)
Comment (bdemann) [reply examples/key_value_store/canisters/rust/Cargo.toml:12]: Yes, the perf result uses candid:CandidType

New line added

File: examples/key_value_store/canisters/motoko/main.mo (hunk 4 above)
Comment (bdemann) [reply examples/key_value_store/canisters/motoko/main.mo:30]: Nice, making that an optional also drastically simplifies the function body :smile:

**Pull request merged**